cmake/codecheck/rules/include_cstdlib.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python -tt
+
+"""Checks that cstdlib is included for some of its functions that we use
+commonly.
+
+Older Macs that build with libc++ instead of libstc++ need this.
+"""
+
+import re
+
+FUNCTION_REGEX = re.compile(
+    r"""(\s|std::)(abs|ato[fil]|atoll|strto[dfl]|strtol[ld]|strtoul[l]{0,1}|std::[s]{0,1}rand|malloc|getenv)\(""")
```

Comment:
```suggestion
    r"""((\s|std::|\(|\[)(abs|ato[fil]|atoll|strto[dfl]|strtol[ld]|strtoul[l]{0,1}|malloc|getenv)|(\bstd::[s]{0,1}rand))\(""")
```
This will catch a little more, for example `(strtol(foo) + bar)`